app/views/layouts/_navbar.html.erb

```diff
@@ -23,6 +23,7 @@
         <li class="<%= li_active_class(crosswalk_issues_partials_path) %>"><%= link_if_not_active 'Crosswalk Partial Matches', crosswalk_issues_partials_path %></li>
         <li class="<%= li_active_class(crosswalk_issues_orphans_path) %>"><%= link_if_not_active 'Ipeds Orphans', crosswalk_issues_orphans_path %></li>
         <li class="<%= li_active_class(archives_path) %>"><%= link_if_not_active 'Archives', archives_path %></li>
+        <li class="<%= li_active_class(calculator_constants_path) %>"><%= link_if_not_active 'Calculator Constants', calculator_constants_path  %></li>
```

Comment:
Now that https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/config/initializers/common.rb#L18 is available should this be wrapped in a `if !production?` check?

Reply:
See dev channel on slack.

Reply:
No.